core/src/test/java/org/jclouds/proxy/ProxyForURITest.java

```diff
@@ -41,12 +42,14 @@
 
    private class MyProxyConfig implements ProxyConfig {
       private boolean useSystem;
+      private boolean canUseProxyFromJvm;
       private Type type;
       private Optional<HostAndPort> proxy;
       private Optional<Credentials> credentials;
 
-      MyProxyConfig(boolean useSystem, Type type, Optional<HostAndPort> proxy, Optional<Credentials> credentials) {
+      MyProxyConfig(boolean useSystem, boolean canUseProxyFromJvm, Type type, Optional<HostAndPort> proxy, Optional<Credentials> credentials) {
          this.useSystem = useSystem;
+         this.canUseProxyFromJvm = canUseProxyFromJvm;
```

Comment:
[minor] Will clean this up to match the naming of the property during the merge, if that's OK with you @ahgittin @nacx...